e2e/tests/functional/notification.e2e.spec.js

```diff
@@ -37,3 +38,42 @@ test.describe('Notifications List', () => {
         // Verify that the other notifications are still present in the notifications list
     });
 });
+
+test.describe('Notification Overlay Regression Test', () => {
```

Comment:
Can we rename this "Notification Overlay" ?

Reply:
Sure!